operators/multiclusterobservability/manifests/base/observatorium/operator.yaml

```diff
@@ -23,37 +23,40 @@ spec:
       affinity:
         podAntiAffinity:
           preferredDuringSchedulingIgnoredDuringExecution:
-          - podAffinityTerm:
-              labelSelector:
-                matchExpressions:
-                - key: app.kubernetes.io/name
-                  operator: In
-                  values:
-                  - observatorium-operator
-              topologyKey: kubernetes.io/hostname
-            weight: 30
-          - podAffinityTerm:
-              labelSelector:
-                matchExpressions:
-                - key: app.kubernetes.io/name
-                  operator: In
-                  values:
-                  - observatorium-operator
-              topologyKey: topology.kubernetes.io/zone
-            weight: 70
+            - podAffinityTerm:
```

Comment:
Any reason why the yaml is being formatted differently?

Reply:
There were some formatting inconsistencies before (or at least my eyes struggled with the formatting), so I ran the files I needed to change through `yamllint` as I found it hard to make changes correctly otherwise.